forkpi_proxy/records/models.py

```diff
@@ -27,3 +27,25 @@ class Meta:
     name = models.TextField(null=False, blank=False, unique=True)
     phone_number = models.CharField(max_length=20, default="", null=True, blank=True)
     is_active = models.BooleanField(default=True)
+
+
+class AppConfigManager(models.Manager):
```

Comment:
We don't want to duplicate the business logic. You can either customize class creation using metaclasses or you can simply use a mixin like below. Please the mixin in a util file so that it can be imported in both environments. Make sure that it only includes code that can run in both Python versions.

```python
class SingletonMixin:
    def get_singleton(self):
        pass

# python 3.3
class AppConfigManager(models.Manager, SingletonMixin):
    pass

# python 3.10
class AppConfigManager(models.Manager, SingletonMixin):
    pass
```